src/tbtcv2-rewards/rewards.ts

```diff
@@ -413,6 +420,11 @@ export async function calculateRewards() {
   // console.log("operatorsData: ", JSON.stringify(operatorsData, null, 4));
```

Comment:
We can delete this console.log (line 420) about `operatorsData` since this data is actually written in the rewards details JSON.

Reply:
Nevermind, It's a minor comment and I didn't realize that the PR is already merged.

Reply:
The post-merge review is still super useful; we can collect this sort of stuff and address it in follow-up PRs 🎉